UnityProjects/MRTKDevTemplate/Assets/Scripts/SampleSceneHandMenu.cs

```diff
@@ -54,7 +54,11 @@ private void Awake()
             previousSceneButton.enabled = IsSceneValid(SceneManager.GetActiveScene().buildIndex - 1);
             nextSceneButton.enabled = IsSceneValid(SceneManager.GetActiveScene().buildIndex + 1);
 
+#if UNITY_2022_1_OR_NEWER
+            SimpleProfiler profiler = Object.FindFirstObjectByType<SimpleProfiler>(FindObjectsInactive.Include);
+#else
             SimpleProfiler profiler = Object.FindObjectOfType<SimpleProfiler>(true);
+#endif
```

Comment:
I haven't checked exactly how many variants there are throughout, but I wonder if it's worth extracting out static (extension or otherwise) methods to allow the `#if` statements to live in only one place. It would also help reduce the number of asmdef changes.

For example, something like (no guarantees on code correctness here 😅)

```c#
public static class MRTKObject
{
    public static T FindObjectByType<T>(bool includeInactive)
    {
#if UNITY_2022_1_OR_NEWER
        return Object.FindFirstObjectByType<T>(includeInactive ? FindObjectsInactive.Include : FindObjectsInactive.Exclude);
#else
        return Object.FindObjectOfType<T>(includeInactive);
#endif
    }
}

Reply:
This seems much cleaner! Will refactor

Reply:
Then this will be a minor version change update to core

Reply:
I've made this change and created another PR https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/515